sound/soc/sof/topology.c

```diff
@@ -138,14 +138,16 @@ static int sof_keyword_dapm_event(struct snd_soc_dapm_widget *w,
 				  struct snd_kcontrol *k, int event)
 {
```

Comment:
Nit: commit subject should say: "fix possible null dereference"

Reply:
right, but I'll squash this so nbd.